pkg/operator/status.go

```diff
@@ -32,6 +34,18 @@ func (optr *Operator) syncVersion() error {
 		return nil
 	}
 
+	previousVersions := co.Status.Versions
+	newVersions := optr.vStore.GetAll()
+
+	if !optr.vStore.Equal(previousVersions) {
+		mco := &corev1.ObjectReference{
+			Kind: co.Kind,
+			Name: co.Name,
+			UID:  types.UID(co.GetUID()),
+		}
+		optr.eventRecorder.Eventf(mco, corev1.EventTypeNormal, "OperatorVersionChanged", fmt.Sprintf("clusteroperator/machine-config-operator version changed from %v to %v", previousVersions, newVersions))
```

Comment:
Let's say we have an upgrade from A to B.  I think the timeline looks like this:

1. operator updated to B
2. operator starts rolling out operand to B
3. operator finishes rolling out operand to B
4. operator updates status.version to indicate B

I'm interested in a marker event for 1 and 4 so that we can track how long the set of steps took.  Which step does this fire for?

Reply:
Alrighty, can do